getFV3GDASensembles

```diff
@@ -34,7 +41,7 @@ for HH in 00 06 12 18; do
 HPSSDIR=/NCEPPROD/5year/hpssprod/runhistory/rh${YYYY}/${YYYY}${MM}/${YYYY}${MM}${DD}
 
 #make links
-for k in $(seq -w 1 80); do
+for k in $(seq -w 1 30); do
```

Comment:
we need 80 for hybrid analysis.

Reply:
@Ruifang-Li, Thanks. I thought we only needed the 30.